test/intern/functional/TreeCompound.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Test dgrid/selector module</title>
```

Comment:
Oops. We apparently have a couple of copypasta'd `title` tags in the functional tests.  I need to do a lint sweep anyway so I'll hit the others then.  I've fixed this one on my branch though.